epochtalk/epochtalk
# Mod log

**Test Procedure:**

1) Ensure you "DELETE FROM roles", checkout core-pg mod-log branch, and run new migrations

2) Log in as a moderator or admin

3) Perform various moderation actions (Editing boards, posts, users, threads, etc...)

4) As an admin or above, visit Admin Panel -> Moderation -> Logs

5) Verify that link to objects are working and that the data looks correct

6) Ensure that filtering looks correct.

**Misc Testing:**

1) Ban a user

2) Visit their profile page, there should be a banner at the top indicating that they are banned and for how long

3) Log in as the banned user, ensure that there is a banner in the header indicating that you are banned

4) Unban the user and ensure that the banners are no longer present
